docs/terms/layer.md

```diff
@@ -1,6 +1,13 @@
-page_title: Layers
-page_description: Organizing the Docker Root File System
-page_keywords: containers, lxc, concepts, explanation, image, container
+<!--[metadata]>
++++
+draft = true
```

Comment:
Should all these documents have "draft=true"? (saw it on a couple, might be a "copy paste" error)